x11-wm/emwm/emwm-1.0a.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit optfeature
+
+DESCRIPTION="Enhanced Motif Window Manager (MWM fork). Supports Xinerama, Xrandr, UTF-8/Xft, and relevant Extended Window Manager Hints (EWMH)."
+HOMEPAGE="http://fastestcode.org/emwm.html"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/alx210/emwm.git"
+else
+	SRC_URI="https://fastestcode.org/dl/emwm-src-${PV}.tar.xz -> ${P}.tar.xz"
+	KEYWORDS="~amd64"
+fi
+
+LICENSE="LGPL-2.1+"
+SLOT="0"
+
+BDEPEND="
+	x11-libs/motif
+"
+
+RDEPEND="
+	${BDEPEND}
+	>=x11-libs/libX11-1.6.2
+	>=x11-libs/libXext-1.3.2
+	x11-libs/libXinerama
+	x11-libs/libXrandr
+	>=x11-libs/libXt-1.1.4
+"
+
+DEPEND="
+	${RDEPEND}
+	x11-base/xorg-proto
+"
+
+if ! [[ ${PV} == 9999 ]]; then
```

Comment:
This isn't wrong at all, but I prefer it in the if ... block for 9999 up above, as it's nearer SRC_URI and such anyway for which it is closely related.

You are welcome to keep it as-is if you wish.